cli/backup/organizations.go

```diff
@@ -53,26 +52,42 @@ func newOrganizationsListCmd() simplecobra.Commander {
 		Long:  description,
 		WithCFunc: func(cmd *cobra.Command, r *support.RootCommand) {
 			cmd.Aliases = []string{"l"}
+			cmd.PersistentFlags().BoolP("with-preferences", "", false, "when set to true, Attempts to retrieve Orgs Preferences (Warning, this is slow due to Grafana current API design)")
 		},
 		RunFunc: func(ctx context.Context, cd *simplecobra.Commandeer, rootCmd *support.RootCommand, args []string) error {
 			filter := service.NewOrganizationFilter(parseOrganizationGlobalFlags(cd.CobraCommand)...)
-			slog.Info("Listing organizations for context", "context", config.Config().GetGDGConfig().GetContext())
-			rootCmd.TableObj.AppendHeader(table.Row{"id", "organization Name", "org slug ID", "HomeDashboardUID", "Theme", "WeekStart"})
-			listOrganizations := rootCmd.GrafanaSvc().ListOrganizations(filter)
+			includePreferences, _ := cd.CobraCommand.Flags().GetBool("with-preferences")
+
+			if includePreferences {
+				rootCmd.TableObj.AppendHeader(table.Row{"id", "organization Name", "org slug ID", "HomeDashboardUID", "Theme", "WeekStart"})
```

Comment:
There are a few places like this that feel like there's copy/paste and chance for desynchronization. 

Could we do something like the following pseudo-code:

rowHeaders = ["id", "organization Name", "org slug ID"]
if includePreferences {
    rowHeaders.append(["HomeDashboardUID", "Theme", "WeekStart"])
}
rootCmd.TableObj.AppendHeader(table.Row{rowHeaders})

Reply:
Fixed